File: de.unistuttgart.iste.sqa.mpw.modeling.generator/src-gen/de/unistuttgart/hamster/command/impl/RemoveEntityCommandImpl.java
package de.unistuttgart.hamster.command.impl;

import de.unistuttgart.hamster.command.RemoveEntityCommand;

public class RemoveEntityCommandImpl extends RemoveEntityCommand {

	public RemoveEntityCommandImpl() {

	}

	@Override
	public void execute() {
		throw new RuntimeException("not implemented");
	}

	@Override
	public void undo() {
		throw new RuntimeException("not implemented");
	}

	@Override
	public void redo() {
		throw new RuntimeException("not implemented");
	}

}

File: de.unistuttgart.iste.sqa.mpw.karasimulator.cpp/core/src/impl/RemoveEntityCommandImpl.cpp
#include "RemoveEntityCommandImpl.h"

#include <stdexcept>

namespace commands {

RemoveEntityCommandImpl::RemoveEntityCommandImpl() {
}

void RemoveEntityCommandImpl::execute() {
	throw std::runtime_error("not implemented");
}

void RemoveEntityCommandImpl::undo() {
	throw std::runtime_error("not implemented");
}

void RemoveEntityCommandImpl::redo() {
	throw std::runtime_error("not implemented");
}

}

